.github/workflows/run_checks.yaml

```diff
@@ -154,10 +154,10 @@ jobs:
           echo "tot_hash=$TOT_HASH" >> $GITHUB_OUTPUT
         continue-on-error: true
 
-      - name: Check warn rebase
+      - name: Report successful application
         if: ${{ steps.apply-baseline.outputs.apply_baseline == 'true' && steps.apply-tip-of-tree.outputs.apply_tot == 'false' }}
```

Comment:
It should warn if it doesn't apply to tip of tree.
The if statement and the actual logic don't match here.

Reply:
Misunderstood the previous comment on this section. I'll update